website/js/components/insilicopred/ProteinLevelSubtile.js

```diff
@@ -0,0 +1,125 @@
+/*eslint-env browser */
+/*global require: false, module */
+'use strict';
+
+import React from "react";
+import classNames from "classnames";
+import {isNumeric} from "../../util";
+
+const mapValToColor = (v) => {
+    if (v < 0.25) {
+        return "bracket_lt25";
+    }
+    else if (v < 0.5) {
+        return "bracket_lt50";
+    }
+    else if (v < 0.75) {
+        return "bracket_lt75";
+    }
+    else {
+        return "bracket_lte100";
+    }
+};
+
+const context = {
+    missense: {
+        keyDomainScores: [
+            { name: 'C0', value: 0.03 },
+            { name: 'C15-C25', value: 0.29 },
+            { name: 'C35-C55', value: 0.66 },
+            { name: 'C65', value: 0.81 },
+            ],
+        irrelevant: 0.02
+    },
+    silentVal: 0.02,
+    nonsenseGrayZonVal: 0.5,
+    nonsenseVal: 0.99
+};
+
+function labelClasses(v) {
+    return classNames('label-field borderless ', mapValToColor(v));
+}
+
+function rowClasses(v, probability) {
+    // FIXME: needs logic to disambiguate cases where the probability matches multiple rows
```

Comment:
is this a must fix now or fix at some point kind of situation?

Reply:
We're still missing the condition that disambiguates the two possible matches for protein-level prob == 0.02, documented in issue #816. This issue's apparently waiting on #831 and #832 before it can be resolved.

Visually this will currently appear as both rows being highlighted. I don't know whether that's acceptable for letting people take a look at the data, or if we should hold off until we've resolved #816.